bokehjs/src/coffee/core/ui_events.coffee

```diff
@@ -130,13 +130,13 @@ export class UIEvents
         if view?
           if view.model.cursor?
             cursor = view.model.cursor()
-          if active_inspectors?
```

Comment:
`gulp test` tests something (slightly) different than bokeh.js bundle in a very different environment than a web browser, so that's not unexpected, though I'm not sure if this is the case here (I would need a closer look).